### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by paluh**
Backlinking to purescript/purescript#3942

### CODE REVIEW

**Comment by JordanMartinez** [comment]
This is blocked by `minibench` (a test dependency), which is also blocked by `lcg`/`globals`.

**Review by JordanMartinez — APPROVED** [review]
CI builds. Need 1 more approval before we can merge this.

**Review by hdgarrood — APPROVED** [review]

**Pull request merged**
